source/02_algo/imitation_learning_en.rst

```diff
@@ -0,0 +1,81 @@
+Imitation Learning
+==================
+
+Problem Definition and Research Motivation
+----------
+
+Imitation Learning (IL) refers to a learning method in which an agent extracts knowledge by learning some expert data, and then replicates the behavior of these expert data. Due to the inherent characteristics of IL,
+It faces two major problems: it requires a lot of training data, and the quality of the training data must be good. In order to solve the above problems, in general, IL can be divided into three directions: IRL (inverse reinforcement learning), BC (behavioral cloning),
+Adversarial Structured IL. The following is a brief analysis of each direction:
+
+
+
+Research Direction
+--------
+
+BC
+~~~~~~~~
+
+BC was first proposed in [1], which proposes a supervised learning method, which directly establishes the state-action mapping relationship by fitting expert data.
+
+The biggest advantage of BC is that it is very efficient and the algorithm is simple, but once the agent encounters a state it has never seen before, it may make the wrong behavior - this problem is called "state drift". In order to solve this problem, the DAgger [2] method adopts a method of dynamically updating the data set, and continuously adds new expert data to the data set according to the real state encountered by the training policy. In the follow-up research, IBC [3] adopted the method of implicit behavior cloning, the key of which is to train a neural network to accept observations and actions, and output a number, which is low for expert actions high for non-expert actions, thus turning behavioral cloning into an energy-based modeling problem.
+
+The current research hotspots of BC algorithms mainly focus on two aspects: meta-learning and behavior cloning using VR devices.
+
+
+IRL
+~~~~~~~~
+
+The main goal of IRL is to solve the problem of finding enough high-quality data during data collection. Specifically, IRL first learns a reward function from expert data, and then uses this reward function for subsequent RL training. With such an approach, IRL can theoretically outperform expert data.
```

Comment:
Inverse reinforcement learning (IRL) is the problem of inferring the reward
function of an agent, given its policy or observed behavior. Specifically, IRL first learns a reward function from expert data, and then uses this reward function for subsequent RL training. IRL can theoretically outperform expert data.